MAUI/Shimmer/getting-started.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the getting started with Syncfusion .NET MAUI Shimmer (SfShimmer) control, its elements and more.
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Getting Started of .NET MAUI Shimmer (SfShimmer)
+
+This section explains how to add the Shimmer control.
+
+## Creating an application using the .NET MAUI Shimmer
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfShimmer` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core), and then install it.
+
+3. To initialize the control, import the control namespace `Syncfusion.Maui.Shimmer` in XAML or C# code.
+
+4. Initialize `SfShimmer.`
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3 5" %}
+
+<ContentPage   
+    . . .
+    xmlns:shimmer="clr-namespace:Syncfusion.Maui.Shimmer;assembly=Syncfusion.Maui.Core">
+
+    <shimmer:SfShimmer />
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1 9 10" %}
+
+using Syncfusion.Maui.Shimmer;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfShimmer shimmer = new SfShimmer();
+        this.Content = shimmer;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
```

Comment:
The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.

Reply:
Comments modified